crates/ironrdp-server/src/encoder/mod.rs

```diff
@@ -28,8 +32,47 @@ impl UpdateEncoder {
     }
 
     pub(crate) fn bitmap(&mut self, bitmap: BitmapUpdate) -> Option<UpdateFragmenter<'_>> {
+        // let len = loop {
+        //     match self.bitmap.encode(&bitmap, self.buffer.as_mut_slice()) {
+        //         Err(e) => match e.kind() {
+        //             ironrdp_pdu::PduErrorKind::NotEnoughBytes { .. } => {
+        //                 self.buffer.resize(self.buffer.len() * 2, 0);
+        //                 debug!("encoder buffer resized to: {}", self.buffer.len() * 2);
+        //             }
+
+        //             _ => {
+        //                 debug!("bitmap encode error: {:?}", e);
+        //                 return None;
+        //             }
+        //         },
+        //         Ok(len) => break len,
+        //     }
+        // };
+
+        // Some(UpdateFragmenter::new(UpdateCode::Bitmap, &self.buffer[..len]))
```

Comment:
Note: leftover which can be removed before merging.

Reply:
fixed